experimental/dds/tree2/src/codec/README.md

```diff
@@ -0,0 +1,15 @@
+# Codec
+
+This library defines types and constructs for organizing code that works with persisted data.
+Its fundamental building block is a codec--a bit of code capable of transcoding between some encoded and in-memory format.
+Typically, for compatibility reasons, aspects of `SharedTree` should work with `ICodecFamily`s (which can do the above for multiple supported formats).
+
+## JSON Validators
+
+Codecs can define draft 6 JSON schema for their encoded data.
+
+One policy choice is whether or not encoded data should be validated, and if so--how.
+Since some `SharedTree` users might not want to pay the bundle-size cost to perform this validation in all environments,
+This policy is injectable using `ICodecOptions` by providing a `JsonValidator`.
```

Comment:
```suggestion
Since some `SharedTree` users might not want to pay the bundle-size cost to perform this validation in all environments,
this policy is injectable using `ICodecOptions` by providing a `JsonValidator`.
```